src/webloader.js

```diff
@@ -115,7 +115,7 @@ function checkInstanceCount(evolv) {
 
 function checkLazyUid(script) {
 	if (script.dataset.evolvLazyUid) {
-		if (!script.dataset.evolvUid) {
+		if (!script.dataset.evolvUid || !isValidGaClientId(script.dataset.evolvUid)) {
```

Comment:
Check if the `data-evolv-lazy-uid` attribute is present
If present then:
1. if no uid is passed, run ga integration
2. if the uid that is passed is not in the right format, run ga integration